Dockerfile

```
@@ -0,0 +1,22 @@
+FROM openjdk:11
+WORKDIR /app
+LABEL org.opencontainers.image.source=https://github.com/gravwell/ocsf-translation-tools
+LABEL org.opencontainers.image.title=ocsf-cli
+LABEL org.opencontainers.image.documentation=https://github.com/gravwell/ocsf-translation-tools/blob/main/README.md
+LABEL org.opencontainers.image.description=Provides a container of the OCSF CLI used to translate source data into the OCSF format
```

```suggestion
LABEL org.opencontainers.image.description="Provides a container of the OCSF CLI used to translate source data into the OCSF format"
```

`docker build` seems to require quotes on this line

---

Hey thanks for pointing this out.  The Dockerfile was failing for you because it trying to copy from the gradle build folder which is not present in the repository. I moved the schema.json to the root of this repository, so you can still build the container without running gradle. 

I also updated the README to explain this situation.